.github/workflows/static_checks.yml

```diff
@@ -92,3 +92,18 @@ jobs:
       - name: Bazel lint check
         run: |
           bash /home/runner/work/oppia-android/oppia-android/scripts/buildifier_lint_check.sh $HOME
+
+  script_checks:
+    name: Script Checks
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
```

Comment:
This seems to be an action of an individual developer. Are we comfortable adding that to our repo? @anandwana001 @BenHenning

Reply:
I think it's good to use it, we already use this at multiple places and everything is working fine till now. Also, no security issues.